docs/src/main/docbook/client.xml

```diff
@@ -999,4 +999,139 @@ Client client = ClientBuilder.newBuilder().sslContext(sslContext).build();</prog
 
 
     </section>
+
+    <section>
+        <title>New Features</title>
+        <para>
+            Jersey 2.30 comes with multiple new features, and this post describes three new interfaces on the client.
```

Comment:
"comes with multiple new features" is not good enough for the documentation. The documentation needs to mention the exact features. Maybe I would create 
- Section, InvocationInterceptors: 
   `<para> Suppose a case that the start of the request is to be logged and even measured....</para>`
- Section InvocationBuilderListener. 

These two sections would be on the same level as New Features. 

Drop `Jersey 2.30 comes... They are...` and start with `Suppose`.